### ISSUE TITLE
Decreased thiamine pyrophosphate

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11642]
**Preferred term label:**
Decreased thiamine pyrophosphate in blood

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C3280874

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
TPK1-Related Thiamine Metabolism Dysfunction Syndrome.  https://www.ncbi.nlm.nih.gov/books/NBK623293/ states that the reduction is present in this disease but absent in Biotin-thiamine-responsive basal ganglia disease

**Your nano-attribution (ORCID)**
